generative/maisi/configs/config_maisi.json

```diff
@@ -0,0 +1,166 @@
+{
+    "quality_check_args": null,
```

Comment:
HI @Can-Zhao , can we put the bundle into model zoo?

Reply:
Hi @yiheng-wang-nv , according to the discussion we had with @daguangxu last Friday (6/28), The model will not be in model-zoo.

Large files such as the weights will be downloaded from NGC catalog (public).

Reply:
As discussed, it is not planned yet.

Reply:
I see, let me resolve it.